skbeam/core/utils.py

```diff
@@ -1115,7 +1115,8 @@ def twotheta_to_q(two_theta, wavelength):
     Returns
```

Comment:
I can't reach the line I want here. Can you note that two_theta here is assumed to be in radians?